tracing-attributes/src/lib.rs

```diff
@@ -546,7 +537,12 @@ impl InstrumentArgs {
                      \"debug\", \"info\", \"warn\", or \"error\", or a number 1-5"
                 )
             },
-            None => quote!(tracing::Level::INFO),
+            None => quote! {
+                compile_error!(
+                    "no verbosity level specified, expected one of \"trace\", \
+                     \"debug\", \"info\", \"warn\", or \"error\", or a number 1-5"
+                )
+            },
```

Comment:
This seems fine to me!

I think it _might_, alternatively, be possible to modify [`InstrumentArgs::parse`](https://github.com/tokio-rs/tracing/blob/27b28773513696220d7dca0d800fe25da67b0f68/tracing-attributes/src/lib.rs#L589-L648) to return a parse error when no level is found...but that might be a bigger change, and I'm not sure if there would be any real benefit. So I think this approach is fine.